package.json

```
@@ -48,9 +48,11 @@
     "test": "lerna run --stream test",
     "test.watch": "lerna run --stream test.watch",
     "ci-build-no-rust": "lerna run --ignore @temporalio/core-bridge --stream build",
+    "ci-per-change-load": "node ./packages/test/lib/load/all-in-one.js --server-address $TEMPORAL_TESTING_SERVER_URL --iterations 10000 --otel-url='' --max-cached-wfs 500",
     "ci-load": "node ./packages/test/lib/load/run-all-stress-ci-scenarios.js --server-address $TEMPORAL_TESTING_SERVER_URL --iterations 10000 --otel-url='' --max-cached-wfs 500",
```

Looks like I forgot to delete all of the args in this command, should be:
```suggestion
    "ci-load": "node ./packages/test/lib/load/run-all-stress-ci-scenarios.js",
```